### PR TITLE
Fan page 2 assignment

### PR DESCRIPTION
**Pull request by brianandreasen1983**
This is the pull request for the fan page 2 assignment with CSS files included.

### CODE REVIEW

**Comment by brianandreasen1983** [comment]
Mary please see the "Re-established branch/re-fork to correct issues". There are times where I have been having to work from multiple computers saving the corrected versions to one drive and have just discovered that my git hub repo is locally stored when I thought it was mapped to one drive. My mistake.

File: index-alt.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Reflex an Arena FPS game</title>
+<meta charset="utf-8">
+<link rel="stylesheet" href="style-alt.css">
```
Comment (mbMosman) [inline index-alt.html]: Alt page links to alt css :heavy_check_mark:

File: index-alt.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <a href="index.html">Home</a> &nbsp; 
+        <a href="http://reflexfps.net/">Reflex Homepage</a> &nbsp;
+        <a href="1vs1.html"> 1 versus 1</a> &nbsp; 
+        <a href="ctf.html"> Capture The Flag</a> &nbsp; 
+        <a href="tdm.html">Team Deathmatch</a> &nbsp; 
+        <a href="weapons.html">Weapons of Reflex</a> &nbsp; 
+        <a href="http://store.steampowered.com/app/328070/">Purchase Reflex on Steam</a> &nbsp;
+    </nav>
+<main>
+    <p> &nbsp; Reflex is a first-person shooter video game that focuses on competitive and casual arena style combat. Arena style combat is different than modern shooters found in Counter-Strike:Global Offensive, Call of Duty or even Halo. It doesn't pertain to any realism and is based on fantasy and imaginative concepts drawn from other games such as Quakeworld and Challenge Promode Arena aka. CPMA which is a Quake III Arena modification.<br></p>
```
Comment (mbMosman) [inline index-alt.html]: We hadn't learned yet how to indent paragraphs using CSS, but watch for this in the upcoming reading.

File: index.html [hunk 3]
```diff
@@ -1 +1,45 @@
-
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Reflex an Arena FPS game</title>
+<meta charset="utf-8">
+<link rel="stylesheet" href="style.css">
```
Comment (mbMosman) [inline index.html]: main page links to main css :heavy_check_mark:

File: style.css [hunk 4]
```diff
@@ -0,0 +1,7 @@
+body {background-color: #000000; color: #ffffff}
```
Comment (mbMosman) [inline style.css]: Alt style uses different colors :heavy_check_mark:

File: style-alt.css [hunk 5]
```diff
@@ -0,0 +1,8 @@
+body {background-color: #ffffff; color: #000000}
```
Comment (mbMosman) [inline style-alt.css]: Alt style uses different colors :heavy_check_mark:

File: style.css [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+h1 {margin-bottom: 10px}
+h1,h2, h3 {color: orangered}
```
Comment (mbMosman) [inline style.css]: alternate heading text color :heavy_check_mark:

File: style.css [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+nav {background-color: #00ff3b}
+p {color: #ff0000}
+.list {color: yellow}
+#copy {color:fuchsia}
```
Comment (mbMosman) [inline style.css]: CSS applied using both style class and id selector :heavy_check_mark:

**Comment by mbMosman** [comment]
Looks like you've got plans for your page :grinning: 

Nice work!

**Pull request closed without merging**